### ISSUE TITLE
Unable to compile due to clutter-gst-2.0 dependency (only 3.0 available) on Ubuntu 24.04

### ISSUE THREAD

**Issue by bertvandepoel** [linuxmint/xplayer#173]
```
 * Xplayer version: git master (21 July 2024)
 * Distribution - Ubuntu 24.04
```

**Issue**

Unable to compile xplayer due to dependency on clutter-gst-2.0 but only 3 is available:
```
bert@berts-desktop:~$ aptitude search clutter | grep gst
p  gir1.2-clutter-gst-3.0 - GObject introspection data for Clutter GStreamer elements
v  gir1.2-cluttergst-3.0 - 
v  gir1.2-cluttergst-3.0-dev - 
p  gstreamer1.0-clutter-3.0 - Clutter PLugin for GStreamer 1.0
p  libclutter-gst-3.0-0 - Open GL based interactive canvas library GStreamer elements
p  libclutter-gst-3.0-dev - Open GL based interactive canvas library GStreamer elements (development files)
p  libclutter-gst-3.0-doc - Open GL based interactive canvas library GStreamer elements (documentation
```

It also complains about gsettings-desktop-schemas even though that's instaled.

**Steps to reproduce**

- clone master repo on Ubuntu 24.04
- clean up autoconf file since it gave me issues: libtoolize --force; aclocal; autoheader; automake --force-missing --add-missing; autoconf
- ./configure
- install missing depencies until you get stuck

**Expected behaviour**

For it to just compile and work with Ubuntu 24.04

**Other information**

Based on https://blog.linuxmint.com/?p=4675 I was under the impression that there was a general aim to maintain X-Apps to have independent options. I'm a big fan of the old totem player before it was all changed and messed up, so xplayer was a godsend to me that I've been using for 6 years now on Ubuntu 18.04. As I'm required to start using Ubuntu 24.04 on a new machine, I was really hoping to get xplayer compiled myself, as no apt/ppa/deb options seemed to be available to me. I've tried many other players but the adaptive scrubscrolling (for lack of a better term) is really a feature I can't go without!

**Comment by leigh123linux** [comment]
Your issue makes no sense, xplayer uses version 3

https://github.com/linuxmint/xplayer/blob/master/configure.ac#L135

https://github.com/linuxmint/xplayer/blob/master/debian/control#L23

**Comment by bertvandepoel** [comment]
Oh now I found it. It just gives a really weird error when it can't find 3.0-dev:

```
checking for clutter-gst-3.0... no
configure: ** Using clutter-gst-2.0
checking for gstreamer-1.0 >= 1.6.0 gstreamer-base-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-pbutils-1.0 gstreamer-tag-1.0 >= 0.11.93 gstreamer-audio-1.0 gstreamer-video-1.0 clutter-1.0 >= 1.10.0 clutter-gst-2.0 >= 1.5.5 clutter-gtk-1.0 cairo gsettings-desktop-schemas... no
configure: error: Package requirements (gstreamer-1.0 >= 1.6.0 gstreamer-base-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-pbutils-1.0 gstreamer-tag-1.0 >= 0.11.93 gstreamer-audio-1.0 gstreamer-video-1.0 clutter-1.0 >= 1.10.0 clutter-gst-2.0 >= 1.5.5 clutter-gtk-1.0 cairo gsettings-desktop-schemas) were not met:

Package 'clutter-1.0', required by 'virtual:world', not found
Package 'clutter-gst-2.0', required by 'virtual:world', not found
Package 'clutter-gtk-1.0', required by 'virtual:world', not found
Package 'gsettings-desktop-schemas', required by 'virtual:world', not found
```

But that's gone now that I've installed 3.0-dev.

However, the problem with gsettings-desktop-schemas remains. Should I file another bug for that?

```
checking for clutter-gst-3.0... yes
configure: ** Using clutter-gst-3.0
checking for gstreamer-1.0 >= 1.6.0 gstreamer-base-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-pbutils-1.0 gstreamer-tag-1.0 >= 0.11.93 gstreamer-audio-1.0 gstreamer-video-1.0 clutter-1.0 >= 1.10.0 clutter-gst-3.0 >= 2.99.2 clutter-gtk-1.0 cairo gsettings-desktop-schemas... no
configure: error: Package requirements (gstreamer-1.0 >= 1.6.0 gstreamer-base-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-pbutils-1.0 gstreamer-tag-1.0 >= 0.11.93 gstreamer-audio-1.0 gstreamer-video-1.0 clutter-1.0 >= 1.10.0 clutter-gst-3.0 >= 2.99.2 clutter-gtk-1.0 cairo gsettings-desktop-schemas) were not met:

Package 'gsettings-desktop-schemas', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables BACKEND_CFLAGS
and BACKEND_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

**Comment by bertvandepoel** [comment]
Oops, nevermind. I uninstalled a bunch of dev libraries after the attempt it seems. I messed around with old totem code and xplayer code until waaaaay too late and got quite confused it seems. 

Now stuck on also needing to compile https://github.com/linuxmint/xplayer-plparser and https://github.com/linuxmint/xapp but I guess I'll have to deal with that. I wish there was an easier way to get this working on Ubuntu 24.04 than manual compiling but I guess that's what I get for really liking adaptive scrollscrubbing!
